src/main/java/core/basesyntax/Excalator.java

```diff
@@ -0,0 +1,4 @@
+package core.basesyntax;
+
+public class Excalator extends Machine {

```

Comment:
The class name 'Excalator' seems to be a typo. It should be 'Excavator' according to the task description. 😊